database/index.js

```diff
@@ -0,0 +1,57 @@
+var mongoose = require('mongoose');
+mongoose.connect('mongodb://localhost/carouselvideo', {useNewUrlParser: true});
+
+var db = mongoose.connection;
+
+var Schema = mongoose.Schema;
+var carousel = new Schema({
+  user : { id: {type: Number, unique: true}, username: String},
```

Comment:
I like that use of 'unique' -- I need to look at my code for that -- thanks!

Reply:
:)